bundles/statistics/statsgrid2016/components/Datatable.js

```diff
@@ -399,6 +400,11 @@ Oskari.clazz.define('Oskari.statistics.statsgrid.Datatable', function (sandbox,
             log.debug('Indicator parameter changed! ');
             me._handleRegionsetChanged();
         });
+        this.service.on('StatsGrid.ClassificationChangedEvent', function (event) {
+            if (event.getChanged().hasOwnProperty('fractionDigits')) {
+                me._handleRegionsetChanged();
```

Comment:
Maybe we should rename _handleRegionsetChanged() to something _updateWholeUI() or something like that? But we can do it later.